plugins/modules/copr.py

```diff
@@ -255,6 +265,10 @@ def _enable_repo(self, repo_filename_path, repo_content=None):
         """
         if not repo_content:
             repo_content = self._download_repo_info()
+        if self.ansible_module.params.get("includepkgs"):
+            repo_content = repo_content.rstrip('\n') + '\nincludepkgs={0}\n'.format(self.ansible_module.params['includepkgs'])
+        if self.ansible_module.params.get("excludepkgs"):
```

Comment:
```suggestion
        if self.ansible_module.params["excludepkgs"]:
```

Reply:
yup will update this , checked it was not necessary